lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -12,7 +12,6 @@
       ->fields('t')
       ->execute()
       ->fetchAll();
-    db_truncate('dosomething_rogue_failed_task_log')->execute();
```

Comment:
do items still get removed from the failed log if they make it to Rogue successfully?